cmd/kube-proxy/app/server_others.go

```diff
@@ -108,14 +108,7 @@ func newProxyServer(
 	iptInterface = utiliptables.New(execer, protocol)
 	kernelHandler = ipvs.NewLinuxKernelHandler()
 	ipsetInterface = utilipset.New(execer)
-	canUseIPVS, err := ipvs.CanUseIPVSProxier(kernelHandler, ipsetInterface)
-	if string(config.Mode) == proxyModeIPVS && err != nil {
-		klog.Errorf("Can't use the IPVS proxier: %v", err)
-	}
-
-	if canUseIPVS {
-		ipvsInterface = utilipvs.New(execer)
```

Comment:
This actually opens up a netlink socket so we shouldn't initiated it if it's not needed.